### PR TITLE
flesh out http identifier documentation

### PR DESCRIPTION
**Pull request by wmorgan**
Try and clarify some things around identifiers.

### CODE REVIEW

File: linkerd/docs/protocol-http.md [hunk 1]
```diff
@@ -85,9 +110,11 @@ routers:
     port: 5000
 ```
 
-A request to `:5000/true/love/waits.php` will be identified as
-`/custom/prefix/true/love` and will be forwarded with `/waits.php` as the URI.
-
+With this configuration, a request to `:5000/true/love/waits.php` will be
+mapped to `/http/true/love` and will be routed based on this name by the
+corresponding dtab. Additionalyl, because `consume` is true, after routing,
```
Comment (adleong) [inline linkerd/docs/protocol-http.md]: typo: Additionalyl

File: linkerd/docs/protocol-http.md [hunk 2]
```diff
@@ -27,56 +32,76 @@ routers:
...
+`methodAndHost` identifier. Using a different identifier would require a
+different set of dtab rules.)
+
 <a name="protocol-http-identifiers"></a>
 ## HTTP/1.1 Identifiers
 
-Identifiers are objects responsible for creating logical names from an incoming
-request with the following parameters:
+Identifiers are responsible for creating logical names from an incoming
+request; these names are then matched against the dtab. HTTP/1.1 identifiers
```
Comment (adleong) [inline linkerd/docs/protocol-http.md]: maybe link to https://linkerd.io/doc/0.7.2/routing/

File: linkerd/docs/protocol-http.md [hunk 3]
```diff
@@ -27,56 +32,77 @@ routers:
...
 
-### Method and Host
+### The Method and Host Identifier
 
-`io.l5d.methodAndHost`
+This identifier is selected by setting the *kind* value of the *identifier*
+configuration block to `io.l5d.methodAndHost`.
 
-HTTP requests are routed by a combination of Host header, method, and URI.
+With this identifier, HTTP requests are turned into glogical names using a
```
Comment (klingerf) [inline linkerd/docs/protocol-http.md]: typo here: glogical => logical

**Comment by klingerf** [comment]
⭐ ✒️  Huge improvement!

**Comment by olix0r** [comment]
:star:

**Pull request merged**